content/about/agreements.md

```diff
@@ -86,7 +87,7 @@ Feedback for jovo can be provided directly in the #jovofeedback slack channel, i
 We agree to be at the lab several hours per day, any day that JHU is open, and respond to reasonable requests from other lab members within one working day, unless otherwise specified.
 Our experience dictates that in person communication and happenstance meetings dramatically increase our productivity.  At the same time, many of us need some time to think or write in quiet, often in isolation. Therefore, we have found that a balance of the two is optimal for everyone (so far).  The particular distribution of time alone vs. with the team is idiosyncratic per person. To respect the individual with the goals of the team we each agree to:
 
-- to be around the lab *many* working hours each day (approximately 10am to 5pm Monday to Friday on all days JHU is open),
+- to be around the lab *many* working hours each day (approximately 10:30am to 6:30pm Eastern Time, Monday to Friday on all days JHU is open),
```

Comment:
I feel like the working hours each day should be ~9-6:30 w/ no expectation that people work that whole time, but an understanding that those are valid hours during which one can meet expectations by being in lab during the day. 10:30 works for some but others definitely get in at like 8.